substrate/frame/revive/src/exec.rs

```diff
@@ -1532,6 +1535,10 @@ where
 		}
 	}
 
+	fn origin(&self) -> Origin<T> {
+		self.origin.clone()
+	}
```

Comment:
Why not return a reference?

Reply:
Copy paste from `fn caller` which returns a value too. But I don't see why it can't be a reference. Changed it